docs.json

```diff
@@ -19,7 +20,8 @@
             "allow-uri-read",
             "toc=left",
             "java=/usr/bin/java",
-            "graphvizdot=/usr/bin/dot"
+            "graphvizdot=/usr/bin/dot",
+            "stackscli_version={{ BUILDNUMBER }}"
         ]
     }
 }
```

Comment:
Is this file ours, if so we should add a newline

Reply:
Fixed in latest push